tests/test_telemetry.py

```diff
@@ -43,6 +42,7 @@ def test_telemetry_proxy_enrichment(self):
     @missing_feature(library="java")
     @missing_feature(library="dotnet")
     @missing_feature(library="python")
+    @missing_feature(library="nodejs")
     def test_telemetry_message_has_datadog_container_id(self):
```

Comment:
This test should pass but doesn't. I've investiguated a bit and the way the Nodejs lib gets the container id is by fetching the cgroup in `/proc/self/cgroup` and running and few regexes on it. But it turns out the cgroup value when running system tests is `0::/`, so the library can't extract any container id.
@cbeauchesne it looks like all libraries skip this test, maybe because the test cannot work.

Reply:
ok, then you can skip it enterily with 

```
@irrelevant(True, reason="cgroup in weblog is 0::/, so this test can't work")
```

Hoping we'll find a way to test this